src/renderer/features/contacts/ContactFilter/model/__tests__/contact-filter.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { allSettled, fork } from 'effector';
+
+import { filterModel } from '../contact-filter';
+import { contactModel } from '@entities/contact';
+
+const mockContact1 = {
+  id: 1,
+  name: 'name 1',
+  address: '222',
+  accountId: '0x333',
+  matrixId: '@444',
+};
+
+const mockContact2 = {
+  id: 5,
+  name: 'name 6',
+  address: '777',
+  accountId: '0x888',
+  matrixId: '@999',
+};
+
+describe('features/contacts/model/contact-filter-model', () => {
+  const scope = fork({
```

Comment:
`scope` must be unique in every test
The whole point of `fork` - it creates isolated scope

> Creates an isolated instance of application. Primary purposes of this method are SSR and testing.
[From Doc](https://effector.dev/en/api/effector/fork/)

Reply:
This way there is no need to call `allSettled(filterModel.events.formInitiated, { scope });` in every test (especially without `await` which is required)

Reply:
Makes sense thanks. I find the duplication of the exact same `fork` somewhat harder to read, because it feels like the scope would be different between tests. But I understand it may help convey the idea that all test is based on an entirely new one.